src/platform/forms/tests/save-in-progress/actions.unit.spec.js

```diff
@@ -393,12 +373,7 @@ describe('Schemaform save / load actions:', () => {
     it('dispatches a not-found if the api returns an empty object', () => {
       const thunk = fetchInProgressForm(VA_FORM_IDS.FORM_10_10EZ, {});
       const dispatch = sinon.spy();
-      global.fetch.returns(
-        Promise.resolve({
-          ok: true,
-          json: () => ({}), // Return an empty object
-        }),
-      );
+      setFetchJSONResponse(global.fetch.onCall(0), {});
```

Comment:
In some of the tests you're simply adding a `url` to an existing `global.fetch.returns` call, but in others you're using the new `setFetchJSONResponse`. What determines which one of these gets used?

Reply:
Are you talking about places where I have code that looks like this instead of using `setFetchJSONResponse`?
```
    it('should dispatch auth error', done => {
      global.fetch.returns({
        catch: () => ({
          then: fn =>
            fn({ ok: false, status: 401, json: () => Promise.resolve() }),
        }),
      });
```

In these cases, `setFetchJSONResponse` (even when I tried altering it to accept a status code as a param) was not returning the correct response to trigger the test to pass, so I left them as is. I am open to other suggestions, but I was trying to balance standardizing our use of fetch mocking with altering the tests as little as possible.

If that is not what you were talking about, could you point me to an example?

Reply:
That's what I was talking about. It sounds like it will be obvious to anyone writing a new test or modifying an existing one which pattern to use, since one of them simply won't work. That's the only reason I was wondering.